### ISSUE TITLE
Multiple withStyles only rendering the first

### ISSUE THREAD

**Issue by Ehesp** [kriasoft/isomorphic-style-loader#37]
I asked this on Gitter originally, but I'm struggling to get the following to work (example):

```
import appStyles from './App.scss`;
import otherStyles from '../OtherComponent/otherStyles.scss';

...

export default withStyles(App, appStyles, otherStyles);
```

This doesn't throw any errors, and both the styles are within the [`styles` array](https://github.com/kriasoft/isomorphic-style-loader/blob/master/src/withStyles.js#L25), only the `appStyles` are loaded in.

Is this the correct way of going about this?

**Comment by Ehesp** [comment]
Note: After upgrading to version `1.0.0`, with a different syntax this problem still occurs.

**Comment by frenzzy** [comment]
Your context insertCss function should handle all style arguments rather than just one, for example:
```js
// client.js
const context = {
  insertCss(...styles) { styles.forEach(style => style._insertCss()); },
};
```
```js
// server.js
const css = [];
const context = {
  insertCss(...styles) { styles.forEach(style => css.push(style._getCss())); },
};
```

**Comment by koistya** [comment]
I guess, we need to create `<StylesContext>` (or `<StylesProvider>`) React component, that can be used to set `insertCss` context variable..

**Comment by Ehesp** [comment]
Right makes sense, we were only taking the first item rather than the array of items. Cheers.

**Comment by adrianricardo** [comment]
@Ehesp what was the resolution?
